source/_redirects

```diff
@@ -2355,3 +2355,4 @@
 /components/sensor.arduino /more-info/removed-integration 301
 /components/switch.arduino /more-info/removed-integration 301
 /integrations/arduino /more-info/removed-integration 301
+/integrations/ee_brightbox /more-info/removed-integration 301
```

Comment:
`ee_brightbox` has more occurrences in this redirect file that needs to be moved to this location and redirect directly to the removed integration page.

Reply:
This should be better